packages/react-ui-ag/src/Filters/PriceFilterCard.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classname from 'classnames'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+import { parseArgs } from '@rentpath/react-ui-utils'
+import { RangeSlider } from '@rentpath/react-ui-core'
+import FilterCard from './FilterCard'
+
+@themed(/^PriceFilter/)
+
+export default class PriceFilterCard extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    title: PropTypes.string,
+    description: PropTypes.string,
+    onApplyClick: PropTypes.func,
+    onCancelClick: PropTypes.func,
+    priceSlider: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.func,
+      PropTypes.object,
+    ]),
+  }
+
+  static defaultProps = {
+    theme: {},
+    title: 'Filter by price',
+    description: 'Set a maximum price for your search.',
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: null,
+    }
+  }
+
+  @autobind
+  handlePriceSliderChange(value) {
+    this.setState({
+      value,
+    })
+  }
+
+  @autobind
+  handleApplyClick() {
+    const { onApplyClick } = this.props
+
+    if (onApplyClick) onApplyClick(this.state.value)
+  }
+
+  @autobind
+  handleCancelClick() {
+    const { onCancelClick } = this.props
+
+    if (onCancelClick) onCancelClick(this.state.value)
+  }
+
+  renderPriceSlider() {
+    const { priceSlider } = this.props
+    const [Slider, props] = parseArgs(priceSlider, RangeSlider)
+    return (<Slider {...(props || {})} onChangeComplete={this.handlePriceSliderChange} />)
```

Comment:
Can you experiment with removing the parentheses and the `(props || {})`? I think in the end they weren't needed.